css/main.scss

```diff
@@ -2480,6 +2480,12 @@ footer {
   }
 }
 
+.team-row {
+  display: flex;
+  flex-direction: row;
```

Comment:
@AliciadW you can actually shorthand this by `flex-flow: row wrap`. :)

Reply:
Thanks @schalkventer I wasn't aware of that. :-)